api/src/policies/users-policy.ts

```diff
@@ -32,15 +32,18 @@ export class UsersPolicy extends BasePolicy<User> {
   }
 
   permittedAttributes(): Path[] {
-    return [
-      "email",
-      "firstName",
-      "lastName",
-      "position",
-      {
-        groupMembershipAttributes: ["departmentId", "divisionId", "branchId", "unitId"],
-      },
-    ]
+    if (this.user.roleTypes.includes(RoleTypes.SYSTEM_ADMIN)) {
+      return [
+        "email",
+        "firstName",
+        "lastName",
+        "position",
+        {
+          groupMembershipAttributes: ["departmentId", "divisionId", "branchId", "unitId"],
+        },
+      ]
+    }
+    return ["email", "firstName", "lastName"]
```

Comment:
I would generally write this as

```ts
permittedAttributes(): Path[] {
  const attributes: Path[] = ["email", "firstName", "lastName", "position"]

  if (this.user.isSystemAdmin) {
    attributes.push({
      groupMembershipAttributes: ["departmentId", "divisionId", "branchId", "unitId"],
    })
  }

  return attributes
}
```

But I don't really have a good explanation, so you are free to leave as is.